### ISSUE TITLE
Uncoordinated ELB vs. RollingUpdate causes service outage

### ISSUE THREAD

**Issue by nelsonjr** [kubernetes/kubernetes#75647]
**What happened**:

When an application with an ELB is upgraded, even with RollingUpdate an outage happens.

**What you expected to happen**:

No service interruption: Before existing pods gets recycled for new pods be registered with ELB, and old pods are removed from ELB.

**How to reproduce it (as minimally and precisely as possible)**:

1. Create a service with `type: LoadBalancer` (I used AWS ELB)
2. Deploy an application to the cluster
3. Confirm the application is running (ensure you're getting 200 code below):
    ```
    while [ 1 ]; do curl -m 1 -s -w " - %{http_code}\n" http://your-elb/; sleep 1; done
    ```
4. Perform an application update using `rolling update`
5. Notice that application will stop responding, likely returning `000` as code
6. After a while -- when ELB catches up -- application starts responding again

**Anything else we need to know?**:

**Environment**:
- Kubernetes version (use `kubectl version`):
```
Client Version: version.Info{Major:"1", Minor:"12", GitVersion:"v1.12.2", GitCommit:"17c77c7898218073f14c8d573582e8d2313dc740", GitTreeState:"clean", BuildDate:"2018-10-30T21:39:38Z", GoVersion:"go1.11.1", Compiler:"gc", Platform:"darwin/amd64"}
Server Version: version.Info{Major:"", Minor:"", GitVersion:"v1.9.1+a0ce1bc657", GitCommit:"a0ce1bc", GitTreeState:"clean", BuildDate:"2018-04-11T20:47:54Z", GoVersion:"go1.9.2", Compiler:"gc", Platform:"linux/amd64"}
```

- Cloud provider or hardware configuration: `AWS`
- OS (e.g: `cat /etc/os-release`): 
```
NAME="CentOS Linux"
VERSION="7 (Core)"
ID="centos"
ID_LIKE="rhel fedora"
VERSION_ID="7"
PRETTY_NAME="CentOS Linux 7 (Core)"
ANSI_COLOR="0;31"
CPE_NAME="cpe:/o:centos:centos:7"
HOME_URL="https://www.centos.org/"
BUG_REPORT_URL="https://bugs.centos.org/"

CENTOS_MANTISBT_PROJECT="CentOS-7"
CENTOS_MANTISBT_PROJECT_VERSION="7"
REDHAT_SUPPORT_PRODUCT="centos"
REDHAT_SUPPORT_PRODUCT_VERSION="7"
```

- Kernel (e.g. `uname -a`):
```
Linux ip-10-68-48-157.ap-southeast-1.compute.internal 3.10.0-862.3.2.el7.x86_64 #1 SMP Mon May 21 23:36:36 UTC 2018 x86_64 x86_64 x86_64 GNU/Linux
```
- Install tools:
- Others:

**Comment by nelsonjr** [comment]
@kubernetes/sig-aws-bugs

**Comment by mattjmcnaughton** [comment]
@nelsonjr thanks for reporting!

If I'm reading your bug report correctly, it looks like you are using v1.9.1. This minor version is no longer supported. As a first step in debugging, could you please upgrade to a supported version? Kubernetes supports the last three minor release versions (i.e. 1.11, 1.12, 1.13 - although 1.14 is being released soon). (As a side benefit, updating will address some CVE's for which I'm not sure fixes have been backported to 1.9.1).

If the problem persists after the update, I'd be happy to chat some more :)
